### ISSUE TITLE
Consistency in return type/value of isPrime(x)

### ISSUE THREAD

**Issue by a-r-r-o-w** [Legrandin/pycryptodome#640]
### Inconsistent return values in `isPrime`

Implementation of [isPrime()](https://github.com/Legrandin/pycryptodome/blob/99273ea596aa6fb4ae9f18f7efa17d6bb359f35b/lib/Crypto/Util/number.py#L341) returns one of `True`, `False`, `0` or `1`.

Minimal reproducible example:

```py
from Crypto.Util.number import *

x = 2 ** 1025

print(isPrime(2))
print(isPrime(3))
print(isPrime(x - 1))
print(isPrime(x - 2))
```

The result is as follows:

```
True
1
0
False
```

I find it slightly bothering in the way it looks on larger outputs, but it isn't much of an issue. Thought it would be good to put this into notice though :sweat_smile: 

### Inconsistent return values and documentation in `_rabinMillerTest`

Documentation of [_rabinMillerTest](https://github.com/Legrandin/pycryptodome/blob/99273ea596aa6fb4ae9f18f7efa17d6bb359f35b/lib/Crypto/Util/number.py#L160) is as follows:

```
    """_rabinMillerTest(n:long, rounds:int, randfunc:callable):int
    Tests if n is prime.
    Returns 0 when n is definitely composite.
    Returns 1 when n is probably prime.
    Returns 2 when n is definitely prime.

    If randfunc is omitted, then Random.new().read is used.

    This function is for internal use only and may be renamed or removed in
    the future.
    """
```

The implementation never returns the value `2`. The return values `0` and `1` and could be changed to `False` and `True` respectively to remove the return value inconsistency in `isPrime` as it calls `_rabinMillerTest` internally.

**Comment by a-r-r-o-w** [comment]
I have opened PR #641 as proposed fix to this issue.

**Comment by Legrandin** [comment]
Fixed with 477dbd338618e24e28265f652426b71d3bbcd21e

**Comment by a-r-r-o-w** [comment]
Looks good!
